extensions/file/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,8 @@
+module.name=file
+
+artifact.export.classPackages=org.mule.extension.file.api,\
```

Comment:
Agree. This is used in order to make the test pass. I created a jira to improve the functional test case to avoid this redundant information. (https://www.mulesoft.org/jira/browse/MULE-9738)